x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/template/LifecyclePolicyConfig.java

```diff
@@ -53,6 +56,21 @@
     public LifecyclePolicyConfig(String policyName, String fileName) {
         this.policyName = policyName;
         this.fileName = fileName;
+        this.templateVariables = Collections.emptyMap();
```

Comment:
We could probably just invoke

```suggestion
        this(policyName, fileName, Collections.emptyMap());
```

Here, so all the logic is in one constructor